mmselfsup/datasets/transforms/formatting.py

```diff
@@ -68,9 +67,20 @@ def transform(self,
             if not isinstance(img, List):
                 img = [img]
             for i, img_ in enumerate(img):
-                if len(img_.shape) < 3:
-                    img_ = np.expand_dims(img_, -1)
-                img_ = np.ascontiguousarray(img_.transpose(2, 0, 1))
+                # to handle the single channel image
+                img_ = np.expand_dims(img_, -1) \
+                    if len(img_.shape) == 2 else img_
+
+                if len(img_.shape) == 3:
+                    img_ = np.ascontiguousarray(img_.transpose(2, 0, 1))
+                elif len(img_.shape) == 5:
+                    # for video data from mmaction with the shape
+                    # (M, C, T, H, W), M = num_crops x num_clips
```

Comment:
```suggestion
                    # (B, C, T, H, W)
```

Reply:
Since currently in MMSelfSup, we only have only one crop and one clip for each video